src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -49,4 +49,20 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
    */
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] =
```

Comment:
I dunno, I think it's okay the way it is... I personally don't feel the `type` would help